packages/opentelemetry-core/src/trace/instrumentation/BasePlugin.ts

```diff
@@ -37,19 +37,19 @@ export abstract class BasePlugin<T> implements Plugin<T> {
   protected _logger!: Logger;
   protected _internalFilesExports!: { [module: string]: unknown }; // output for internalFilesExports
   protected readonly _internalFilesList?: PluginInternalFiles; // required for internalFilesExports
-  protected _config!: PluginConfig;
+  protected _config: PluginOptions = {};
 
   enable(
     moduleExports: T,
     tracer: Tracer,
     logger: Logger,
-    config?: PluginConfig
+    config?: PluginOptions
   ): T {
     this._moduleExports = moduleExports;
     this._tracer = tracer;
     this._logger = logger;
     this._internalFilesExports = this._loadInternalFilesExports();
-    if (config) this._config = config;
+    this._config = config || {};
```

Comment:
If you pass {} by default, we don't need to do `this._config = {};` in concrete class (inside the Ctor)